### DISCUSSION TITLE
Installing GraphQL plugin breaks Strapi

### DISCUSSION THREAD

**Discussion by jonathanpuc** [strapi/strapi#5495] [Legacy Topics]
**Describe the bug**
When installing a new version of Strapi and installing the graphql plugin via in the root folder via `strapi install graphql` and trying to run a strapi command such as `strapi develop`, the following error occurs:

```
[2020-03-12T09:46:23.200Z] debug ⛔️ Server wasn't able to start properly.
[2020-03-12T09:46:23.202Z] error Error: Cannot find module 'jws'
```

**Steps to reproduce the behavior**
1. Install new Strapi instance
2. Install Graphql plugin via Strapi
3. Try run `strapi develop` command

**Expected behavior**
Running `strapi develop` should spin up the CMS.

**System**
- Node.js version: 12.9.0 also tried with 10.19.0
- NPM version: 6.13.4
- Strapi version: 3.0.0-beta.19.3
- Database: N/A
- Operating system: Mac OS

**Comment by lauriejim** [comment]
Hello! Tried to reproduce, but it works on my side.

Here is what I did - https://www.loom.com/share/17597d0f666646df85cd948b5c0e82e2
